main.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+)
+
+func main() {
+	var address string
+	var requestsNumber int64
+	var timeoutMilliseconds float64
+	requests := newRequests()
+	err := initVariables(&address, &requestsNumber, &timeoutMilliseconds)
```

Comment:
u can use init() func for flags, it helps to remove a lot of logic from your code

Reply:
https://golang.org/doc/effective_go.html#init